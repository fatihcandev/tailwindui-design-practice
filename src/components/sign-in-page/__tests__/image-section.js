import React from 'react'
import renderer from 'react-test-renderer'

import ImageSection from '../image-section'

describe('ImageSection', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ImageSection />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
