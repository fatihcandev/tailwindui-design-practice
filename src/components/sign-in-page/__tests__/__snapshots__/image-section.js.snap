// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageSection renders correctly 1`] = `
<div
  className="imageSection"
/>
`;
